src/context/SyntheticsStateContext/SyntheticsStateContextProvider.tsx

```diff
@@ -139,6 +143,16 @@ export function SyntheticsStateContextProvider({
 
   const orderEditor = useOrderEditorState(ordersInfo.ordersInfoData);
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const lastWeekPeriod = useMemo(() => getTimePeriodsInSeconds().week, [formatDate(Date.now())]);
```

Comment:
it's not worth to format date per each render to check if week has changed
```
const lastWeekPeriod = useMemo(() => getTimePeriodsInSeconds().week, []);
```
should be enough